TheSufferfest/SwiftOverlays
# Migration to Swift 4.0

#### What's this PR do?
Migration of this pod, developer [peterprokop](https://github.com/peterprokop) added the support for Swift 4.0

#### Any background context you want to provide?
No

#### How should this be manually tested?
Get the latest version of https://github.com/TheSufferfest/SufferfestWorkoutPlayer on your machine, pull from master, open the podfile, change the line that indicates the source of the SwiftOverlays pod for this:
`pod 'SwiftOverlays', :git => 'https://github.com/TheSufferfest/SwiftOverlays.git', :branch => 'swift-4.0'`
Open the Terminal, navigate to the project (cd "MyDisk/MyUser/SufferfestWorkoutPlayer")
run `pod update SwiftOverlays`
Open the project on Xcode, run the app on a device or simulator, look for the places where the overlay is shown, for instance, when the workout player is paused, all the overlays must be shown and work as always

#### Could any automated tests be added for this?
No

#### What are the relevant JIRA issues?
This is part of the effort to migrate Sufferfest Workout player to Swift 4.0
https://thesufferfest.atlassian.net/browse/IA-1064

#### Screenshots (if appropriate)
None

#### Open issues
No
